RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py

```diff
@@ -65,18 +66,22 @@
         tdcSaturation = hfnoseDigitizer.digiCfg.feCfg.tdcSaturation_fC,
         tdcOnset      = hfnoseDigitizer.digiCfg.feCfg.tdcOnset_fC,
         toaLSB_ns     = hfnoseDigitizer.digiCfg.feCfg.toaLSB_ns,
-        fCPerMIP      = cms.vdouble(1.25,2.57,3.88) #100um, 200um, 300um
+        fCPerMIP      = fCPerMIP_mpv
         ),
 
     algo = cms.string("HGCalUncalibRecHitWorkerWeights")
 )
 
 from Configuration.Eras.Modifier_phase2_hgcalV10_cff import phase2_hgcalV10
-phase2_hgcalV10.toModify( HGCalUncalibRecHit.HGCEEConfig , fCPerMIP = fCPerMIP_v10 ) 
-phase2_hgcalV10.toModify( HGCalUncalibRecHit.HGCHEFConfig , fCPerMIP = fCPerMIP_v10 )
+phase2_hgcalV10.toModify( HGCalUncalibRecHit.HGCEEConfig , fCPerMIP = fCPerMIP_mean ) 
+phase2_hgcalV10.toModify( HGCalUncalibRecHit.HGCHEFConfig , fCPerMIP = fCPerMIP_mean )
+
+from Configuration.Eras.Modifier_phase2_hgcalV16_cff import phase2_hgcalV16
```

Comment:
You should add a new `Configuration.Eras.Modifier_phase2_hgcalV16_cff`, at the moment is missing and the test is failing 

```
File "/cvmfs/cms-ci.cern.ch/week1/cms-sw/cmssw/36728/22058/CMSSW_12_3_X_2022-01-27-2300/python/RecoLocalCalo/HGCalRecProducers/HGCalUncalibRecHit_cfi.py", line 79, in <module>
    from Configuration.Eras.Modifier_phase2_hgcalV16_cff import phase2_hgcalV16
ModuleNotFoundError: No module named 'Configuration.Eras.Modifier_phase2_hgcalV16_cff'
```